core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/BillingApiServiceV2.scala

```diff
@@ -64,8 +64,7 @@ trait BillingApiServiceV2 extends UserInfoDirectives {
                       parsedStartDate,
                       parsedEndDate
                     ).map {
-                      case Some(spendReportResults) => StatusCodes.OK -> Option(spendReportResults)
-                      case None => StatusCodes.NotFound -> None
+                      sr => StatusCodes.OK -> sr
```

Comment:
Can you explain what this is doing?

Reply:
On closer inspection, I think this is actually unnecessary and I'll remove it pending passing tests, but I'll explain why I thought it was needed in the first place.

`sr` is an instance of `SpendReportingResults`, and I wanted to make sure that the correct HTTP status code (200 in this case) was returned along with `sr`. `StatusCodes.{code} -> {result}` is one way of specifying the HTTP status code. However, it seems that Akka (or some other Scala magic, I'm not sure which) will default to a 200 and return your result without doing this.

Reply:
I forgot to remove this unnecessary code from this branch, but it'll be in the next spend reporting PR for per-workspace breakdowns